hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/key/KeyUtils.java

```diff
@@ -99,6 +99,7 @@ public static boolean isValid(@Nullable final Key pbjKey) {
             for (Key keys : ((ThresholdKey) key.value()).keys().keys()) {
                 if (!isValid(keys)) {
                     isKeyListValid = false;
+                    break;
```

Comment:
Guess you could also just return `false` here.

Reply:
No, we have this check after break: return (threshold >= 1 && threshold <= length && isKeyListValid);

Reply:
Yes and `isKeyListValid` will be `false`. Therefore we will return `false` anyway.

But it does not really matter. It is just a minor change that has no real effect. Just go ahead and change it in any way you like.